airflow/www/static/js/task-instances.js

```diff
@@ -67,7 +67,8 @@ export default function tiTooltip(ti, {includeTryNumber = false} = {}) {
   if (ti.task_id !== undefined) {
     tt += `Task_id: ${escapeHtml(ti.task_id)}<br>`;
   }
-  tt += `Run: ${formatDateTime(ti.execution_date)}<br>`;
+  tt += `Execution Date: ${formatDateTime(ti.execution_date)}<br>`;
+  tt += `Following Execution Date: ${formatDateTime(ti.following_execution_date)}<br>`;
```

Comment:
If this is a manually triggered dag run , or one with `schedule_interval=None` it doesn't have a following date.

What does this show in those cases?

Reply:
That's a really great question. 

For manual runs, it shows the following execution date, when it should be showing the same execution date.

![screen](https://user-images.githubusercontent.com/7297387/86222278-a269b780-bb4b-11ea-9c1d-ada897ea85db.png)

For schedule interval of type "None", it explodes in a traceback:
```
...
  File "/opt/airflow/airflow/www/decorators.py", line 57, in wrapper
    return f(*args, **kwargs)
  File "/opt/airflow/airflow/www/views.py", line 1415, in tree
    dag_run['following_execution_date'] = dag.following_schedule(execution_date).isoformat()
AttributeError: 'NoneType' object has no attribute 'isoformat'
```

I'll make fixes to both and comment when done.